logdb/sequence.go

```diff
@@ -23,6 +23,12 @@ const (
 	logIndexMask = (1 << logIndexBits) - 1
 )
 
+func init() {
+	if blockNumBits+txIndexBits+logIndexBits > 63 {
+		panic("total bits in sequence exceeds max bit allowed")
+	}
+}
```

Comment:
Given these are consts, should this be in a test ?

Reply:
I don't have a strong opinion on this, it is for someone edited the bits then started from source. Adding to tests will make it clean?

Reply:
I guess my thought process is that if someone modifies the code and doesn't run the tests, it's likely that they might be setting themselves up for bugs.

Reply:
what Pedro said :)

Reply:
Updated!